server/requirements.txt

```diff
@@ -39,4 +39,6 @@ pandas>=1.3.5
 # Downloading the named-entity recognition (NER) library spacy and the large EN model
 # using the guidelines here: https://spacy.io/usage/models#production
 spacy>=3.0.0,<4.0.0
-en_core_web_lg @ https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.4.1/en_core_web_lg-3.4.1-py3-none-any.whl
\ No newline at end of file
+# en_core_web_lg @ https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.4.1/en_core_web_lg-3.4.1-py3-none-any.whl
```

Comment:
Remove this comment if we're no longer using lg.

Reply:
Done